Makefile

```diff
@@ -1,5 +1,6 @@
 OPERATOR_ASPECT_TAG ?= "ghcr.io/syntasso/kratix-cli/from-api-to-operator"
 HELM_ASPECT_TAG ?= "ghcr.io/syntasso/kratix-cli/helm-resource-configure"
+CROSSPLANE_ASPECT_TAG ?= "ghcr.io/syntasso/kratix-cli/from-api-to-crossplane-claim"
```

Comment:
fix: cant' find this image in our registry; will this be pushed by the pipeline?

Reply:
i dont think so. Release processes normally happen on main, so I think running it manually after its merged makes sense. Also it doesn't need to be release until the kratix CLI update is released